src/pages/Metrics/MetricsPage.scss

```diff
@@ -0,0 +1,344 @@
+    @include desktop {
+      margin-right: 48px;
+      width: 100%;
+      max-width: 448px;
+    }
+  }
+
+  &__image {
+    @include noselect;
+
+    width: 100%;
+
+    @include tablet {
+      width: none;
+    }
+  }
+
+  &__metric {
+    display: flex;
+    flex-direction: column;
+    margin-top: 24px;
+    width: 100%;
+
+    @include desktop {
+      margin-top: 0;
+    }
+  }
+
+  &__title {
+    margin-bottom: 16px;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 1.4;
+    text-align: left;
+  }
+
+  &__uniqueness {
+    margin-bottom: 28px;
+  }
+
+  &__uniqueness-bars {
+    display: flex;
+    align-items: center;
+    border-radius: 20px;
+    padding: 16px;
+    box-shadow: 1px 4px 16px rgb(180 173 189 / 15%);
+
+    @include desktop {
+      padding: 16px 24px;
+    }
+  }
+
+  &__stroke-bars {
+    display: flex;
+    flex-direction: column;
+    margin-left: 24px;
+    width: 100%;
+
+    @include desktop {
+      margin-left: 48px;
+    }
+  }
+
+  &__box {
+    display: flex;
+    align-items: flex-start;
+    flex-direction: row;
+    margin-bottom: 16px;
+    width: 100%;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__features-item {
+    margin-right: 8px;
+    margin-bottom: 8px;
+    border-radius: 8px;
+    padding: 8px 16px;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 1.2;
+    background-color: $color-grey-light-100;
+  }
+
+  &__features-box-items {
+    display: flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    margin-top: 12px;
+  }
+
+  &__features-box {
+    margin-bottom: 24px;
+  }
+
+  &__features-box-colors {
+    display: flex;
+    align-items: center;
+    margin-top: 12px;
+  }
+
+  &__features-color {
+    margin-right: 16px;
+    border-radius: 4px;
+    width: 24px;
+    height: 24px;
+  }
+
+  &__subtitle {
+    width: 100%;
+    max-width: 30%;
+    font-weight: 500;
+    font-size: 14;
+    line-height: 1.4;
+    text-align: left;
+    color: $color-dark-grey;
+  }
+
+  &__bar {
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    margin-left: 16px;
+    width: 100%;
+    max-width: 70%;
+  }
+
+  &__bar-background {
+    border-radius: 6px;
+    width: 100%;
+    height: 8px;
+    min-width: 48px;
+    background: $color-grey-light-300;
+  }
+
+  &__bar-progress {
+    border-radius: 6px;
+    height: 8px;
+    background: linear-gradient(90deg, #735be4 0%, #9f8af8 100%);
+  }
+
+  &__bar-percentage-text {
+    margin-left: 4px;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 1.1;
+
+    span {
+      font-size: 12px;
+      color: $color-dark-grey;
+    }
+  }
+
+  &__loader-container {
+    margin-left: 30px;
+    width: 100%;
+    max-width: 70%;
+    text-align: left;
+  }
+
+  &__image-loader-text {
+    position: relative;
+    display: inline-block;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 1.2;
+    color: $color-grey-light-400;
+
+    &::after {
```

Comment:
<img width="465" alt="Screenshot 2023-04-18 at 4 07 49 PM" src="https://user-images.githubusercontent.com/39587602/232759276-8f25c1f2-137f-4776-afdd-80f9b53b0abe.png">

Three dots are detached from the text if it is longer that one line.

Looks like originally they were not in the designs?
If it is approved with the designer, maybe it's worth adding them beginning with a tablet / desktop resolution to avoid such misalignment. Unless you can think of another way without investing too much time into it :)